pyproject.toml

```diff
@@ -25,7 +25,7 @@ packages = [
 [tool.poetry.dependencies]
 python = ">=3.9,<4.0"
 importlib-metadata = {version = "^4.0", python = "<3.9"}
-ansys-fluent-core = "~=0.21.dev1"
+ansys-fluent-core = "~=0.21.0"
```

Comment:
We should set pyfluent dependency to be compatible to the current dev version (~=0.22.dev0) in the main branch of parametric. So that, pyfluent code of all future dev version during 0.22 development (0.22.dev1, 0.22.dev2 etc.) will be automatically exercised in paramteric.

The release version of parametric should depend on release pyfluent version (like ~=0.21.0).

Reply:
Sure. We will update this. Thank you.

Reply:
right @mkundu1, Thank you.